src/openrct2/world/small_scenery.cpp

```diff
@@ -0,0 +1,489 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
+/*****************************************************************************
+* OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+*
+* OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+* For more information, visit https://github.com/OpenRCT2/OpenRCT2
+*
+* OpenRCT2 is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* A full copy of the GNU General Public License can be found in licence.txt
+*****************************************************************************/
+#pragma endregion
+
+#include "../network/network.h"
+
+#include "../rct2.h"
+#include "../network/network.h"
+
+extern "C"
+{
+    #include "footpath.h"
+    #include "scenery.h"
+    #include "map.h"
+    #include "park.h"
+    #include "../cheats.h"
+    #include "../ride/track_design.h"
+}
+
+static money32 SmallSceneryRemove(sint16 x, sint16 y, sint8 baseHeight, uint8 quadrant, uint8 sceneryType, uint8 flags)
+{
+    if (!map_is_location_valid(x, y)) {
+        return MONEY32_UNDEFINED;
+    }
+    money32 cost;
+
+    rct_scenery_entry *entry = get_small_scenery_entry(sceneryType);
+    if (entry == (rct_scenery_entry *)-1)
+    {
+        log_warning("Invalid game command for scenery removal, scenery_type = %u", sceneryType);
+        return MONEY32_UNDEFINED;
+    }
+    cost = entry->small_scenery.removal_price * 10;
+
+    gCommandExpenditureType = RCT_EXPENDITURE_TYPE_LANDSCAPING;
+    gCommandPosition.x = x + 16;
+    gCommandPosition.y = y + 16;
+    gCommandPosition.z = baseHeight * 8;
+
+    if (!(flags & GAME_COMMAND_FLAG_GHOST) && game_is_paused() && !gCheatsBuildInPauseMode) {
```

Comment:
All braces should go on their own lines as per the coding style.
There are more occurrences in this file, but I won't leave a line note at every single instance.